README.md

```diff
@@ -1,20 +1,30 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+-An click handler is a jquery function that allows you to write code that will be executed when the click event occurs
+
+(b) Where in your JS file should you put it, and WHY?
+
+-in `$(document).ready()` so it gets loaded and is waiting to be put to use
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+-You might be using the wrong jQuery reference.
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+
+-Ideally at the bottom, it contains any objects that need to be loaded first. It can go at the top too, though.
```

Comment:
You're right, but the question was WHAT goes in the document ready. You already answered that above - click handlers need to go there.